catalog/app/containers/Admin/Buckets.tsx

```diff
@@ -203,6 +223,48 @@ function SnsField({
   )
 }
 
+const useHintStyles = M.makeStyles((t) => ({
+  icon: {
+    fontSize: '1.125em',
+    marginLeft: t.spacing(0.5),
+    marginTop: -1,
+    opacity: 0.5,
+    verticalAlign: -4,
```

Comment:
This looks a bit hacky. As I see, `vertical-align: middle` gives the same result as marginTop + verticalAlign

Reply:
`vertical-align: middle` results in a different layout:
<img width="310" alt="Screenshot 2021-08-09 at 10 57 31" src="https://user-images.githubusercontent.com/122294/128665178-27a4d148-4cf2-4d6b-ba24-9743a08a8b39.png">

vs layout w/ `marginTop: -1` and `verticalAlign: -4`:
<img width="300" alt="Screenshot 2021-08-09 at 10 57 44" src="https://user-images.githubusercontent.com/122294/128665181-5b32ad34-cf6d-4b46-83c1-3468ac1c10e0.png">